amical/tests/test_io.py

```diff
@@ -41,6 +43,24 @@ def example_bss(global_datadir):
     return bss
 
 
+def test_add_infos_header_commentary():
+    # Make sure that _add_infos_header handles _HeaderCommentaryCards from astropy
+
+    # Create a fits header with commentary card
+    hdr = fits.Header()
+    hdr["HISTORY"] = "History is a commentary card"
+
+    # SimulatedData avoids requiring extra keys in infos
+    infos = munch.Munch(orig="SimulatedData", instrument="unknown")
+
+    # Add hdr to infos laceholders for everything but hdr
+    mf = munch.Munch(pixelSize=1.0)
+    infos = _add_infos_header(infos, hdr, mf, 1.0, "afilename", "amaskname", 1)
+
+    # Convert everything to munch object
+    dict2class(infos)
+
+
```

Comment:
I know we've been building up in this file but `test_io.py` was meant to contain tests for input/outputs only. Can you start a new, more specific test file with this test please ?

Reply:
@neutrinoceros Sure, how should I name the file, and how specific should it be ? I could create something like `test_extract` or `test_bispect`. Some other tests could go in there to because bss is in the `session` scope, if I'm not mistaken ?

Reply:
if you need the bss fixture in other test modules we can simply move it to `conftest.py`, and if the session scope isn't relevant anymore we can simply remove it :)
I don't have a good suggestion for how to name the new file atm, but if you can think of one that _also_ fits some future tests you'd like to add, that's probably the best choice.

Reply:
I moved this the test to `test_extraction.py`.